package.json

```diff
@@ -142,8 +142,8 @@
     "vue-jest": "^3.0.7"
   },
   "engines": {
-    "node": ">=14.0.0 <15.0.0",
-    "npm": "<2.0.0",
+    "node": ">=16.13.0",
```

Comment:
Maybe add a `<17` so that we stay on LTS?

Reply:
@ThibaultNocchi Good catch! 👏